UI/MVUX/src/MVUX/Presentation/FeedViewSample/FeedViewModel.cs

```diff
@@ -0,0 +1,57 @@
+using Uno.Extensions.Reactive;
+
+namespace MVUX.Presentation.FeedViewSample;
+
+public partial record FeedViewModel
+{
+	private readonly Random _random = new();
+
+	public IFeed<Person> DefaultPerson { get; }
+
+	public IFeed<Person> CustomLoadingPerson { get; }
+
+	public IFeed<Person> ErrorPerson { get; }
+
+	public FeedViewModel()
+	{
+		DefaultPerson = Feed.Async(GetRandomPersonAsync);
+
+		CustomLoadingPerson = Feed.Async(GetRandomPersonWithLargerDelayAsync);
+
+		ErrorPerson = Feed.Async(ThrowErrorAsync);
+	}
+
+	private async ValueTask<Person> GetRandomPersonAsync(CancellationToken ct)
+	{
+		await Task.Delay(2000, ct);
+
+		var people = new[]
+		{
+			new Person("Master", "Yoda"),
+			new Person("Darth", "Vader"),
+			new Person("Luke", "Skywalker"),
+		};
```

Comment:
Same for all such initializer that only adds noise in the feed code, move them to static members.